README.md

```diff
@@ -207,6 +207,14 @@ corresponding headers in /workspace/install/include, and the Python
 wheel files in /workspace/install/python. The image will also contain
 the built client examples.
 
+**Important Note:** When running either the server or the client using
```

Comment:
Should this be added to shm docs instead? https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_shared_memory.md

Reply:
I can add a link from the shared memory extension to the note here too. Since the issue relates to both client and server we can mention it in both places.